src/test.rs

```diff
@@ -343,3 +344,20 @@ fn check_test_environment() {
     #[cfg(target_os = "linux")]
     fn test_iface() {}
 }
+
+#[test]
+fn test_get_network_interfaces_ips() {
+    test_iface_ips();
```

Comment:
Okay, yeah I wasn't sure if that was a reasonable expectation. I'll see if I can figure out another way to test this.